app/driver/settings.py

```diff
@@ -110,6 +111,12 @@
     map(int, os.environ.get('DJANGO_POSTGIS_VERSION', '2.1.3').split("."))
 )
 
+# File storage
+DEFAULT_FILE_STORAGE = 'storages.backends.overwrite.OverwriteStorage'
+
+# Primary record label
+PRIMARY_RECORD_LABEL = os.environ.get("DRIVER_PRIMARY_RECORD_LABEL", "Incident")
```

Comment:
If the celery task function is switched to take a `record_type_id` then this and the changes to `deployment/ansible/roles/driver.app/defaults/main.yml` below can be removed.